proof-market/intro.mdx

```diff
@@ -0,0 +1,94 @@
+import WebInterface from '@site/static/img/proof-market-icons/web-interface.png'
+import API from '@site/static/img/proof-market-icons/api.png'
+import Code from '@site/static/img/proof-market-icons/code.png'
+import Class from '@site/static/img/proof-market-icons/class.png'
+import Calculate from '@site/static/img/proof-market-icons/calculate.png'
+import School from '@site/static/img/proof-market-icons/school.png'
+import BugReport from '@site/static/img/proof-market-icons/bug-report.png'
+import Feedback from '@site/static/img/proof-market-icons/feedback.png'
+import LibraryBooks from '@site/static/img/proof-market-icons/library-books.png'
+
+import {Card, CardSection} from '@site/src/components/CardSection'
+
+# `=nil;` Proof Market
+
+## Introduction
+
+Proof Market is a marketplace of zero-knowledge proofs.
+
+Zero-knowledge proofs are powerful but generating them is a compute-heavy task. Proof Market offers an alternative: anyone can post a request for a specific proof and procure said proof after it is generated by a designated producer.
```

Comment:
`and procure said proof`

Now sure what you wanted to write here, some misspell

Reply:
I have rephrased this bit but it was not a misspell: 'said' is a reference word to something that was mentioned previously